idea/data-storage/src/common/helpers/get-meta-hash.ts

```diff
@@ -0,0 +1,13 @@
+import { HexString } from '@polkadot/util/types';
+import { GearProgram } from '@gear-js/api';
+import { GearCode } from '@gear-js/api/Code';
+
+export async function getMetaHash<T extends GearProgram | GearCode>(tx: T, hex: HexString): Promise<null | HexString> {
```

Comment:
What's the point to use generic here?

Reply:
Replaced with created type `GearApiModule` instead of this `genric` type here.